genesyscloud/provider/provider.go

```diff
@@ -257,7 +122,7 @@ func configure(version string) schema.ConfigureContextFunc {
 
 		providerSourceRegistry := getRegistry(&platform, version)
 
-		err := InitSDKClientPool(data.Get("token_pool_size").(int), version, data)
+		err := InitSDKClientPool(context, version, data)
```

Comment:
Won't this var name `context` clash with the package name?

Reply:
Yes, thank you!